src/pages/studyView/tabs/CNSegments.tsx

```diff
@@ -99,41 +121,57 @@ export default class CNSegments extends React.Component<
         return study ? study.referenceGenome : DEFAULT_GENOME;
     }
 
+    get selectionInfo() {
+        if (!this.isSampleCountWithinThreshold) {
+            return 'Too many samples for copy number segmentation view, make a selection on Summary tab first.';
```

Comment:
```suggestion
            return `Too many samples (>${this. isSampleCountWithinThreshold}) for copy number segmentation view, make a selection on Summary tab first.`;
```
maybe good to indicate how many are too many